extensions/BMO/Extension.pm

```diff
@@ -713,6 +713,18 @@ sub bug_format_comment {
         }
     });
 
+    # link github pull requests and issues
+    push (@$regexes, {
+        match => qr#\b([A-Za-z0-9_\.-]+\/[A-Za-z0-9_\.-]+\#[0-9]+)\b#mx,
```

Comment:
Yeah, listing projects, seems to be a great idea. I'll have a look on BMO. Thanks for the review, Dylan!